pkg/recipes/terraform/config/providers/aws.go

```diff
@@ -41,6 +43,7 @@ const (
 	awsRegionParam    = "region"
 	awsAccessKeyParam = "access_key"
 	awsSecretKeyParam = "secret_key"
+	awsRoleARN        = "role_ARN"
```

Comment:
Thanks for figuring this out. Is this the format expected by Terraform? Curious to look at relevant docs if you have that handy.

Reply:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs#assuming-an-iam-role-using-a-web-identity has the information. But I have not yet gotten to the terraform. I added this just so I can get the changes to compile with new model.

Reply:
are you planning on completing it in a follow up PR or as a part of this PR? If a follow up PR is needed, then I'd recommend returning an error message that explicitly states that it is unsupported. Leaving it in half baked state might result into error messages which are difficult to make sense out of.